rpc/handlers.go

```diff
@@ -781,3 +781,27 @@ func getAddInvokeTxCode(err error) int {
 		return jsonrpc.InternalError
 	}
 }
+
+// AddDeployAccountTransaction relays an deploy account transaction to the gateway.
+//
+// It follows the specification defined here:
+// https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_write_api.json#L74
+// Note: No checks are performed on the incoming request since we rely on the gateway to perform sanity checks.
+// As this handler is just as a proxy. Any error returned by the gateway is returned to the user as a jsonrpc error.
```

Comment:
```suggestion
// It follows the [specification].
// Note: This handler is a proxy. No checks are performed on the incoming request since we rely on the gateway to perform sanity checks. Any error returned by the gateway is returned to the user as a jsonrpc error.
//
// [specification]: https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_write_api.json#L74
```
We can use [godoc formatting](https://go.dev/doc/comment) here.

Reply:
Unfortunately, this causes linting errors (on the [spec]:.. line). Also the current version is also consistent with the rest of the doc comments, so I'd prefer to keep it as is for now, if that's okay.